templates/settings.yaml.erb

```diff
@@ -46,6 +46,14 @@
     :type: <%= scope.lookupvar("foreman::logging_type") %>
     :layout: <%= scope.lookupvar("foreman::logging_layout") %>
 
+<% if Array(scope.lookupvar('foreman::cors_domains')).size >= 1 -%>
```

Comment:
It's already guaranteed to be an array. Should it be `<% unless scope.lookupvar('foreman::cors_domains').empty? -%>`

Reply:
Ah, true. Still living in a Puppet 3 world. I'm getting old. 👴